shopfloor_base_default_profile/models/res_user.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Brian McMaster <brian@mcmpest.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    shopfloor_default_profile_ids = fields.One2many(
```

Comment:
Hmm I don't get why you can have multiple profiles on a user. 
If the whole point is to have "a default profile selected for the user" I'd expect to see only one here.